flake.nix

```diff
@@ -14,6 +14,8 @@
     utils,
     rpki-cli,
   }:
+    # Add a kartograf module for NixOS (see module.nix for details)
+    { nixosModules.kartograf = import ./module.nix self; } //
```

Comment:
fine to throw this in the body of the outputs, if you want

Reply:
I applied this kludgy merge approach (which I also use in my own flakes) because I never found a way to get this working with flake-utils `lib.eachDefaultSystem`.

If I put the module in the output body, I end up with

```
git+file:///home/virtu/code/kartograf
├───devShell
│   ├───aarch64-darwin: development environment 'nix-shell'
│   ├───aarch64-linux omitted (use '--all-systems' to show)
│   ├───x86_64-darwin omitted (use '--all-systems' to show)
│   └───x86_64-linux omitted (use '--all-systems' to show)
├───nixosModules
│   ├───aarch64-darwin: NixOS module
│   ├───aarch64-linux: NixOS module
│   ├───x86_64-darwin: NixOS module
│   └───x86_64-linux: NixOS module
└───packages
    ├───aarch64-darwin
    │   ├───default: package 'kartograf-1.0.0'
    │   └───kartograf: package 'kartograf-1.0.0'
    ├───aarch64-linux
    │   ├───default omitted (use '--all-systems' to show)
    │   └───kartograf omitted (use '--all-systems' to show)
    ├───x86_64-darwin
    │   ├───default omitted (use '--all-systems' to show)
    │   └───kartograf omitted (use '--all-systems' to show)
    └───x86_64-linux
        ├───default omitted (use '--all-systems' to show)
        └───kartograf omitted (use '--all-systems' to show)
```

instead of 

```
git+file:///home/virtu/code/kartograf
├───devShell
│   ├───aarch64-darwin: development environment 'nix-shell'
│   ├───aarch64-linux omitted (use '--all-systems' to show)
│   ├───x86_64-darwin omitted (use '--all-systems' to show)
│   └───x86_64-linux omitted (use '--all-systems' to show)
├───nixosModules
│   └───kartograf: NixOS module
└───packages
    ├───aarch64-darwin
    │   ├───default: package 'kartograf-1.0.0'
    │   └───kartograf: package 'kartograf-1.0.0'
    ├───aarch64-linux
    │   ├───default omitted (use '--all-systems' to show)
    │   └───kartograf omitted (use '--all-systems' to show)
    ├───x86_64-darwin
    │   ├───default omitted (use '--all-systems' to show)
    │   └───kartograf omitted (use '--all-systems' to show)
    └───x86_64-linux
        ├───default omitted (use '--all-systems' to show)
        └───kartograf omitted (use '--all-systems' to show)
```

If you know a workaround or were thinking of another place to put it, let me know.

Reply:
ah I see, yea that's a limitation of flake-utils. the merge approach is fine.